cli/src/command/account.rs

```diff
@@ -992,3 +998,36 @@ async fn print_address(account: &Account, address: &AccountAddress) -> Result<()
 
     Ok(())
 }
+
+async fn print_outputs(mut outputs: Vec<OutputData>, title: &str) -> Result<(), Error> {
+    if outputs.is_empty() {
+        println_log_info!("No outputs found");
+    } else {
+        println_log_info!("{title}");
+        outputs.sort_unstable_by(output_booked_timestamp_and_output_id);
+
+        for (i, output_data) in outputs.into_iter().enumerate() {
+            let booked_time = to_utc_date_time(output_data.metadata.milestone_timestamp_booked() as u128 * 1000)?;
+            let formatted_time = booked_time.format("%Y-%m-%d %H:%M:%S UTC").to_string();
+
+            println_log_info!(
+                "{:<5}{}\t{}\t{}\t{}",
+                i,
+                &output_data.output_id,
+                output_data.output.kind_str(),
+                formatted_time,
+                if output_data.is_spent { "Spent" } else { "Unspent" },
+            );
+        }
+    }
+
+    Ok(())
+}
+
+fn outputs_ordering(a: &OutputData, b: &OutputData) -> std::cmp::Ordering {
+    (b.metadata.milestone_timestamp_booked(), a.output_id).cmp(&(a.metadata.milestone_timestamp_booked(), b.output_id))
```

Comment:
Why is this doing (b, a) (a, b) ?

Reply:
it's ordered reversed in time (so that latest timestamps appear first in the list), but ordered forward lexicographically for output ids. So in case the timestamp is the same it will be ("a", ts), ("b", ts), ("c", ts), ...

Reply:
```sh
Outputs:
0    0xf3fabffe29fed6e6fc052dadbf5541db706138332ecae1de226752ff6e0678f70000     Alias   2023-09-07 17:02:55 UTC Unspent
1    0xf3fabffe29fed6e6fc052dadbf5541db706138332ecae1de226752ff6e0678f70100     Foundry 2023-09-07 17:02:55 UTC Unspent
2    0xf3fabffe29fed6e6fc052dadbf5541db706138332ecae1de226752ff6e0678f70200     Basic   2023-09-07 17:02:55 UTC Unspent
3    0xb37041e9aeb03ceeabfde051c07211363d21744340fc03cd444024f86aed2d6f0000     Alias   2023-09-07 17:02:45 UTC Spent
4    0xb37041e9aeb03ceeabfde051c07211363d21744340fc03cd444024f86aed2d6f0100     Basic   2023-09-07 17:02:45 UTC Spent
5    0x79d760938a15171814ed0269266aa836a9ec09ddf74c11c0d81bb9bd05074d8a0000     Nft     2023-09-05 15:53:16 UTC Unspent
6    0x79d760938a15171814ed0269266aa836a9ec09ddf74c11c0d81bb9bd05074d8a0100     Basic   2023-09-05 15:53:16 UTC Spent
7    0x5fbac890c1ac34c580135d885eaa0045c7ecd32d0c97105c9ff6a426502dec070100     Basic   2023-09-05 15:44:42 UTC Spent
```

Reply:
You say `ordered forward lexicographically for output ids` and your screenshot shows decreasing IDs